packages/grid/backend/grid/db/init_db.py

```diff
@@ -13,17 +15,24 @@
 
 
 def init_db(db: Session) -> None:
+    if not len(node.setup):  # Check if setup was defined previously
+        # Build Syft Message
+        msg = CreateInitialSetUpMessage(
+            address=node.address,
+            name="Jane Doe",
+            email=settings.FIRST_SUPERUSER,
+            password=settings.FIRST_SUPERUSER_PASSWORD,
+            domain_name=settings.DOMAIN_NAME,
+            budget=55.55,
+            reply_to=node.address,
+        ).sign(signing_key=node.signing_key)
 
-    # Build Syft Message
-    msg = CreateInitialSetUpMessage(
-        address=node.address,
-        name="Jane Doe",
-        email=settings.FIRST_SUPERUSER,
-        password=settings.FIRST_SUPERUSER_PASSWORD,
-        domain_name=settings.DOMAIN_NAME,
-        budget=55.55,
-        reply_to=node.address,
-    ).sign(signing_key=node.signing_key)
+        # Process syft message
+        _ = node.recv_immediate_msg_with_reply(msg=msg).message
 
-    # Process syft message
-    _ = node.recv_immediate_msg_with_reply(msg=msg).message
+    else:  # Uses the Owner root key to update node keys
+        owner = node.users.first(role=node.roles.owner_role.id)
+        root_key = SigningKey(owner.private_key.encode("utf-8"), encoder=HexEncoder)
```

Comment:
What do you think about moving this process of directly configuring the node to syft instead of grid?